patches/form-data+3.0.1.patch

```diff
@@ -0,0 +1,8 @@
+diff --git a/node_modules/form-data/lib/browser.js b/node_modules/form-data/lib/browser.js
```

Comment:
It's a pretty dirty hack, but I have no other ideas on how to fix it.
It comes from:
https://github.com/cowprotocol/cow-sdk/pull/113/files#diff-a84359e624a8cdb6ef9afe0087de38a8451aa36a5ad8149026c68f9fb799c8e9R6

To make SDK be useful for both web and nodejs I changed order-book `request.ts` to axios.
But, it also added `import FormData from 'form-data'`. 
By default the import lead us to the `/node_modules/form-data/lib/browser.js` file and it doesn't work well with nodejs.

If I build the project as `icrobundle -f modern,esm,cjs --target node`, then the import above lead us to the `node_modules/form-data/lib/form_data.js` file and it works good for nodejs, but doesn't for browser :(

So, my the best solution is this patch.
@alfetopito @anxolin maybe you can suggest smth better

Reply:
I don't have any better ideas. If this work, good with me

Reply:
@anxolin yes, with nodejs 18 this hack is not needed.

>Maybe we can use for node-js in the nodejs app?
Do you mean `node-fetch`? I would like to add it as a polyfill, but there are no options to do it via `microbundle` :(
I have tried to add it just at the beginning of `order-book/api.ts` but it doesn't work as well, because `node_modules/form-data/lib/browser.js` is initialized before.

Reply:
Fixed using https://github.com/lquixada/cross-fetch